plugins/experimental/ssl_session_reuse/src/scope_lock.h

```diff
@@ -0,0 +1,56 @@
+/** @file
+
+  scope_lock.h
```

Comment:
This whole file would go away if you used stdlib <mutex>.

Reply:
Done